client/src/components/game/lab2/Instructions/secondInstructions.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import Button from '../components/header/buttons/button';
+import './secondaryInstructions.css';
+
+/*
+Page containing the instructions for the second popup the user sees
+this instructional popup covers color vision defiencies
+*/
+const SecondInstructions = ({closePage, selectOption, toWhiteBackground,
+  background}) => {
+
+  var alreadyCalled = false;
+  if (!alreadyCalled) {
+    selectOption('Protanopia');
+  }
+
+  if (background !== 'white') {
+    toWhiteBackground();
+  }
+
+  const closeInstructions = () => {
+    closePage();
+  }
+
+  return (
+    <div className="instructionsContainer">
+      <p className='secondInstructionTitle'>Good job so far!</p>
+      <ul>
+        <li className='instructionsItem'>
+          As you can see, this game isn't too difficult. However, to a user with a <em>color
```

Comment:
The width of the instructions on this page seem different than the width on the reading page

Reply:
If I'm not mistaken I think this is because its trying to stay consistent with the game instance. I can look into it if you think I should.

Reply:
Ah that makes sense!